src/asmcnc/apps/drywall_cutter_app/widget_drywall_shape_display.py

```diff
@@ -365,6 +365,7 @@ def __init__(self, **kwargs):
         self.engine = kwargs['engine']
         self.kb = kwargs['kb']
         self.localization = kwargs['localization']
+        self.cs = kwargs['cs']
```

Comment:
kwargs in a Kivy class should be reserved for kivy where possible. this implementation won't work kivy>2.

Reply:
(maybe a ticket for fixing it in the rest of the file)